### PR TITLE
func: bump to 0.24

### PR DESCRIPTION
**Pull request by lance**
Bumps func to 0.24

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: func@0.23.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+require 'fileutils'
+
+class Func < Formula
```
Comment (dsimansk) [inline func@0.23.rb:3]:
```suggestion
class FuncAT023 < Formula
```

Comment (lance) [reply func@0.23.rb:3]: Ahh thank you!

Comment (csantanapr) [reply func@0.23.rb]: This reviews are hard to review inspect by eye, specially the shasum strings. 
There need to be a GitHub action that cares of this for every bump request in addition of adding a e2e test

Comment (lance) [reply func@0.23.rb]: I agree with you Carlos

**Pull request merged**